File: ejemplos-clases-constructores/Ejemplo2/Fabrica/src/fabrica/Contabilidad.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package fabrica;

/**
 *
 * @author USUARIO
 */
public class Contabilidad {

    private int sueldo;
    private int horas;

    public Contabilidad(int c, int b) {
    }

    public void estableceSueldo(int c) {
        sueldo = c;
    }

    public void establacerHoras(int b) {
        horas = b;
    }

    public int obtenerSueldo() {
        return sueldo;
    }

    public int obtenerHoras() {
        return horas;
    }
}

File: ejemplos-objetos/Ejemplo1/PythonProject/Contabilidad.py
# Created on : 29 may 2024, 21:34:58
# Author     : USUARIO


# This is a sample Python script.

# Press Shift+F6 to execute it or replace it with your code.



   class Contabilidad:

    def __init__(self, c, b):
        self.sueldo = c
        self.horas = b

    def establece_sueldo(self, c):
        self.sueldo = c

    def establecer_horas(self, b):
        self.horas = b

    def obtener_sueldo(self):
        return self.sueldo

    def obtener_horas(self):
        return self.horas
